### ISSUE TITLE
feature proposal: Video-Text Snippets

### ISSUE THREAD

**Issue by AdamHibble** [codenetwork/CodeNetworkTV#5]
I am interested in building a image processing service, the purpose of this will be to take an image of a desktop - and  extract code snippets from it. 

I imagine it being used as a service with the video player, being able to copy and paste code that is on the stream. 

Love some input on the endpoint,

We have the option to send via multipart/form-data POST, or Base64 encode the file

Response will be a simple json 
```
{
    status: "OK",
    text: "Code Extracted From Text"
} 
```

**Comment by 25564** [comment]
I know this project is meant to be a learning experience but wouldn't the far more practical solution to have this data streamed from the User by utilising either a plugin or similar local program? The reason would be two fold. Firstly the person streaming does not want the code easily available to the general public and the margin of error would have to be minuscule due to the fragility of code (Simple spelling mistakes could cause fatal errors).

**Comment by AdamHibble** [comment]
Fair point, primary reason for me wanting to do it this way is interest rather than practicality. If someone wants to add the functionality using a plugin that is 100% reasonable, I have no interest in working on it though.

**Comment by 25564** [comment]
Fair enough. Although I feel as if this discussion can be delayed to a later point as we currently have an empty repo but I see your point and it does have the added benefit of being a standalone feature that could be utilised for footage that it was not intended to be used with such as legacy video.
